kafka-tools/generate-downloads/README.md

```diff
@@ -41,3 +43,79 @@ $ make run # same make vars as running in env above
+
+```shell
+dp ssh prod publishing 10
+```
+
+and run this command (substituting the decrypted password where indicated):
+
+```shell
+mongo admin --tls --host prod-docdb-cluster.cluster-cqs87nhaxvnx.eu-west-2.docdb.amazonaws.com --username root --password <Place decrypted docdb_login_password here> --tlsCAFile "/etc/docdb/rds-combined-ca-bundle.pem"
+```
+
+NOTE: If the above command fails -
+
+ 1. Check the cluster address has not changed in AWS console
+
+ 2. If boxes in Prod are rebuilt the numer `10` used above may change and you will need to try other numbers and re-enter the `mongo ...` command until you get the mongo prompt
+
+Once you get this prompt:
+
+`rs0:PRIMARY>`
+
+You can enter for example these two commands (adapt to your needs by adjusting the string `weekly-deaths-local-authority` to the equivalent in the URI provided in the support ticket URI that you have):
+
+```shell
+use datasets
+
+db.getCollection('instances').find({"state":"published", "last_updated":{$gt:new ISODate("2022-08-20T00:00:00")}, "links.dataset.id":/weekly-deaths-local-authority/})
+```
+
+and when this example was run on 25th Aug 2022 it produced two results.
+
+Copy and paste the results into your editor (in this case, vscode and save as `inst.json`, so that we can right click in the document and select `Format Document` to easily identify the fields mentioned below).
+
+From the support ticket URI we copy this sub string: `/datasets/weekly-deaths-local-authority/editions/2022/versions/32` and search for it in the two results.
+
+The first instance of it that we are interested in should be in links.version.href and a little way before that we find the `INSTANCE_ID` as the `"id"` with value of (in this example): `"e581b0c9-8e99-44db-ad71-47fad689827a"` as per:
+
+```json
+    "id": "e581b0c9-8e99-44db-ad71-47fad689827a",
+    "last_updated": ISODate("2022-08-23T08:51:02.456Z"),
+    "e_tag": "16a25b6b85710f7ebbeeaa065eb36ec896af529a",
+    "links": {
+        "dataset": {
+            "href": "http://10.30.154.142:10400/datasets/weekly-deaths-local-authority",
+            "id": "weekly-deaths-local-authority"
+        },
+        "dimensions": {},
+        "edition": {
+            "href": "http://10.30.44.112:10400/datasets/weekly-deaths-local-authority/editions/2022",
+            "id": "2022"
+        },
+        "job": {
+            "href": "http://10.30.154.142:10700/jobs/26627f87-2ee9-467e-b44a-79a0bf679eaf",
+            "id": "26627f87-2ee9-467e-b44a-79a0bf679eaf"
+        },
+        "self": {
+            "href": "http://10.30.155.83:10400/instances/e581b0c9-8e99-44db-ad71-47fad689827a"
+        },
+        "version": {
+            "href": "http://10.30.155.83:10400/datasets/weekly-deaths-local-authority/editions/2022/versions/32",
+            "id": "32"
+        }
+    }
+```
+
+You can follow this process to extract the `"id"` for the `INSTANCE_ID` to use for the issue you are working on in other sections of this document.
```

Comment:
worth adding a reminder to ensure that the password is not left in history ?

Reply:
Instructions adjusted to say use space at start of command.